package.json

```diff
@@ -294,6 +295,34 @@
     "watchify": "^3.11.1",
     "webpack": "^4.41.6"
   },
+  "allowScripts": {
+    "node-sass": true,
+    "optipng-bin": true,
+    "gifsicle": true,
+    "jpegtran-bin": true,
+    "scrypt": true,
+    "chromedriver": true,
+    "geckodriver": true,
+    "@sentry/cli": true,
```

Comment:
Should we specify the version in `package.json` here as an additional precaution? Tedious perhaps, but I'm not sure if it will do exactly what we want with this config. See the following quote from https://www.npmjs.com/package/allow-scripts:

> Running the command will scan the list of installed dependencies (using an existing `package-lock.json` or `npm-shrinkwrap.json` or by creating one on the fly).

Speaking of which, don't we want this to respect `yarn.lock`?

Reply:
If we can resolve the `yarn.lock` concern, there's also this to consider:

Should we specify the version in `package.json` here as an additional precaution? Tedious perhaps, but I'm not sure if it will do exactly what we want with this config.

Reply:
I started with version specification, but then switched them all to true. I figured it would be tedious to maintain the list.

Reply:
What we want is a strict/controlled allow/deny list of dep script execution. The plan is to shift this into LavaMoat at some point.

Reply:
@rekmarks since there is no package-lock it temporarily creates a shrinkwrap from the installed deps. shrinkwrap.json is just a package-lock file. https://docs.npmjs.com/cli/v6/configuring-npm/shrinkwrap-json
it is generated from the installed deps and without network access. completes in 7 sec on my machine
the shrinkwrap is only used to enumerable the package paths in `node_modules`
https://github.com/dominykas/allow-scripts/blob/8deda58d96b8bc7573d63fee0cc843a2797d84b5/lib/index.js#L27
and then identify the ones with install hooks
https://github.com/dominykas/allow-scripts/blob/8deda58d96b8bc7573d63fee0cc843a2797d84b5/lib/index.js#L100-L110
so I dont expect an issue from yarn vs npm

Reply:
> Should we specify the version

hmm, my first response was "no, it doesn't matter"
but now i think that for the enabled packages it would provide some additional protections.
@EtDu can you update the PR to use exact versions for the allowed packages? the disallowed packages do not require exact versions

Reply:
The only drawback here is the need to maintain if we update a dep